polyglot.js

```diff
@@ -248,24 +276,14 @@ class PolyGlot {
             type: Boolean,
             onChange: () => this.updateChatMessages()
         });
-        const orig = Messages.prototype.sayBubble;
-        // TODO: Switch to chatBubble hook once we get the original ChatMessage as part of hook arguments
-        Messages.prototype.sayBubble = (message) => {
-            if (message.data.type == CONST.CHAT_MESSAGE_TYPES.IC) {
-                const lang = message.getFlag("polyglot", "language") || ""
-                if (lang != "" && !this.known_languages.has(lang)) {
-                    try {
-                        // Create a new message so we don't modify the original, since we can't change the html itself
-                        message = new ChatMessage(duplicate(message.data));
-                        message.data.content = message.data.content.replace(this.regexp_letters, this.randomRune);
-                    } catch (err) {
-                        // In case we can't replace it for some reason, just prevent it from appearing
-                        return;
-                    }
-                }
-            }
-            return orig(message);
-        }
+        game.settings.register("polyglot", "salt", {
+            name: "Unique Texts",
+            hint: "Enable this option to make every single text look different.",
+            scope: "world",
+            config: true,
+            default: false,
+            type: Boolean
+        });
```

Comment:
I don't understand the option, and users won't either. Maybe change the name and hint to : 
name: "Randomize runes"
hint: "Enabling this option will cause the scrambled text to appear different every time, even if the same message is repeated"